src/frontend/src/pages/ProjectDetailPage/ProjectEdit/ProjectEditContainer.tsx

```diff
@@ -111,50 +108,41 @@ const ProjectEditContainer: React.FC<ProjectEditContainerProps> = ({ project, ex
     remove: removeConstraint
   } = useFieldArray({ control, name: 'constraints' });
   const { mutateAsync } = useEditSingleProject(project.wbsNum);
+  const [projectManagerId, setprojectManagerId] = useState<string | undefined>(project.projectManager?.userId.toString());
+  const [projectLeadId, setProjectLeadId] = useState<string | undefined>(project.projectLead?.userId.toString());
 
   if (allUsers.isLoading || !allUsers.data) return <LoadingIndicator />;
   if (allUsers.isError) {
     return <ErrorPage message={allUsers.error?.message} />;
   }
 
   const users = allUsers.data.filter((u) => u.role !== 'GUEST');
+  const onSubmit = async (data: any) => {
+    const { name, budget, summary, bomLink, googleDriveFolderLink, taskListLink, slideDeckLink } = data;
+    const rules = data.rules.map((rule: any) => rule.rule || rule);
```

Comment:
why did you change lines 121 and 122, don't think you need to change them for this ticket

Reply:
Line 122 wasn't changed, but I changed line 121 back

Reply:
Also we set the value in the payload, so its redundant?